test/eventinge2e/source_to_ksvc_test.go

```diff
@@ -14,7 +14,7 @@ import (
 
 const (
 	pingSourceName    = "smoke-test-ping"
-	image             = "quay.io/openshift-knative/helloworld-go"
+	image             = "quay.io/openshift-knative-serving-test/helloworld:v1.3"
```

Comment:
In this PR we use `${IMAGE_REGISTRY_NAME}` in other places but here we hardcode it to quay.io. Does it make sense to hardcode it everywhere? What's the chance that the registry will change? I suppose it's close to zero.

Reply:
yes this go file so we didn't directly replace ${IMAGE_REGISTRY_NAME}. I tried out some thing like this ex and it worked so appended in the code as an example
```
package main
import (
        "fmt"
        "os"
)
func main() {
        image:=(os.Getenv("IMAGE_REGISTRY_NAME"))+"/openshift-knative-serving-test/helloworld:v1.3"
    fmt.Println(image)
}

```